www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,84 @@
+var React = require('react/addons');
+var classNames = require('classnames');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      checkedValue: true,
+      uncheckedValue: false
+    };
+  },
+  onChange: function (e) {
+    var val = e.target.checked ? this.props.checkedValue : this.props.uncheckedValue;
+    this.valueLink.requestChange(val);
+  },
+  isChecked: function () {
+    return this.valueLink.value === this.props.checkedValue;
+  },
+  render: function () {
+    var linkState = this.props.linkState;
+    this.valueLink = linkState(this.props.id);
```

Comment:
does this need to be rebound on every `render`? Is this something we can simply do once in `componentWillMount` instead?